content/docs/reference/design-docs/train-sim-v3/_index.en.md

```diff
@@ -81,6 +67,62 @@ TrainSim([train simulator])
 SimResults[simulation result curve]
 ```
 
+### Target schedule
+The target schedule is a list of target arrival times at points specified along the path.
+To respect the schedule, the train may have to not use its maximum traction.
+
+### Train state
+The train state is a vector of properties describing the train at a given point in time.
+- position
+- speed
+- position of pantographs 
+- driver reaction times ?
+- battery state ?
+- time elapsed since the last update
+
+### Driving instructions
+
+{% pageinfo color="warning" %}
+This is a copy paste from the driving instructions subpage.
+{% end %}
+
+Driving instructions model what the train has to do, and under what conditions.
+Driving instructions are generated using domain constraints such as:
+
+- unsignaled line speed limits
+- permanent signaled speed limits
+- temporary speed limits
+- dynamic signaling:
+  - block / moving block
+  - dynamically signaled speed restrictions
+- neutral zones
+- stops
+
+### Path properties
+
+Path properties are the physical properties of the path, namely elevation and curves.
+
+### Driver behavior module
+
+The driver behavior module updates the train state based on:
```

Comment:
To me, the main point of the driver behavior module is to represent both actual real-world driver behavior and margins. Maybe would be nice to give an example of these (e.g. driver anticipates a speed limit?), and state that both are represented with the same abstraction.

I'm not super sure we should get into the details of the inputs/outputs of the driver behavior module here in the high-level definition?

(Take this with a grain of salt, I'm no expert~)

Reply:
I feel like dicussing and pinning what the dreiver bahvior module represents would help us a lot, going forward